shared/actions/teams.js

```diff
@@ -376,6 +376,10 @@ const _getDetails = function*(action: TeamsGen.GetDetailsPayload): Saga.SagaGene
       team: publicity.teamShowcase.isShowcased,
     }
 
+    const teamOperation = yield Saga.call(RPCTypes.teamsCanUserPerformRpcPromise, {
```

Comment:
Does this have to be sync? It might be better to create a new action handler, and then just do a put for the action so this can be handled in a separate function. It would also make this more reusable since if anything else needs to do this, it can just dispatch the action.

Reply:
@MarcoPolo we talked about this in NYC yesterday, I think we landed on factoring this out in 5720 (leave-a-team bug) which is probably the first place we'll need to call this separate from `_getDetails`